install/pre-installation.md

```diff
@@ -361,12 +397,29 @@ These variables will need to be set for many procedures within the CSM installat
    - From Cray using `curl`:
 
       > - `-C -` is used to allow partial downloads. These tarballs are large; in the event of a connection disruption, the same `curl` command can be used to continue the disrupted download.
+      > - CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. Using `http_proxy` or `https_proxy` in any way other than the following examples will cause many failures in subsequent steps.
+
+      Without proxy:
 
       ```bash
       curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
         "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
       ```
 
+      With https proxy:
+
+      ```bash
+      https_proxy=https://example.proxy.net:443 curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+      With http proxy:
```

Comment:
Do we need the http proxy section here? We are giving them a specific URL for this artifact and it is an HTTPS URL. So I think the http_proxy would just be ignored in this case.